### PR TITLE
Add fed and state congressional district nums

### PR DESCRIPTION
**Pull request by wrridgeway**
Just a quick addition of some columns in `vw_pin_universe` that are functions of pre-existing columns.

### CODE REVIEW

**Comment by wrridgeway** [comment]
```
select distinct
    census_data_year,
    census_congressional_district_geoid,
    census_congressional_district_num,
    census_state_representative_geoid,
    census_state_representative_num,
    census_state_senate_geoid,
    census_state_senate_num,
    census_acs5_data_year,
    census_acs5_state_representative_geoid,
    census_acs5_state_representative_num,
    census_acs5_state_senate_geoid,
    census_acs5_state_senate_num
from z_dev_wridgeway_default.vw_pin_universe
```
[output](https://github.com/ccao-data/data-architecture/files/15407344/69a666ea-40ce-4b09-9cb1-e94112d940df.csv)

**Review by dfsnow — COMMENTED** [review]
**issue (blocking):** What's up with the 2011 IDs in the output? We may want to fix those in the source.

**Comment by wrridgeway** [comment]
> **issue (blocking):** What's up with the 2011 IDs in the output? We may want to fix those in the source.

yeah, not sure. i'll take a look. i assumed that it was an expected change in formatting.

**Comment by wrridgeway** [comment]
```
select distinct
    census_data_year,
    census_congressional_district_geoid,
    census_congressional_district_num,
    census_state_representative_geoid,
    census_state_representative_num,
    census_state_senate_geoid,
    census_state_senate_num,
    census_acs5_data_year,
    census_acs5_state_representative_geoid,
    census_acs5_state_representative_num,
    census_acs5_state_senate_geoid,
    census_acs5_state_senate_num
from z_dev_wridgeway_default.vw_pin_universe
```
[new output](https://github.com/ccao-data/data-architecture/files/15437461/ee945105-7393-4cf6-8153-744dd746fd11.csv)

We'll need to trigger a rebuild of location.census after this is merged.

File: etl/scripts-ccao-data-warehouse-us-east-1/spatial/spatial-census.R [hunk 1]
```diff
@@ -59,7 +59,15 @@ normalize_census_geo <- function(key) {
         -ends_with("LSAD", ignore.case = TRUE)
       ) %>%
       rename_with(tolower) %>%
-      mutate(geometry_3435 = st_transform(geometry, 3435)) %>%
+      mutate(
+        geometry_3435 = st_transform(geometry, 3435),
+        geoid =
+          if (key == "spatial/census/congressional_district/2011.geojson") {
```
Comment (wrridgeway) [inline etl/scripts-ccao-data-warehouse-us-east-1/spatial/spatial-census.R:65]: This is pretty brute force, but I'm not sure how else to do it.

File: etl/scripts-ccao-data-warehouse-us-east-1/spatial/spatial-census.R
Comment (dfsnow) [inline etl/scripts-ccao-data-warehouse-us-east-1/spatial/spatial-census.R:1]: **issue (blocking):** Now you've got 'ZZ' geoids in the output.

Comment (wrridgeway) [reply etl/scripts-ccao-data-warehouse-us-east-1/spatial/spatial-census.R:1]: those were there before as well. i can remove them.

**Comment by wrridgeway** [comment]
[Alright, all gone.](https://github.com/ccao-data/data-architecture/files/15445723/8ebf9c82-7785-4d22-8a41-f86eacd68152.csv)

**Comment by wrridgeway** [comment]
Did a quick investigation into which parcels are missing census geoids for 2024:

- 1026 pin10s are missing lat/lon so can't be spatially joined to census
- 22 pin10s are in the lake

image

**Comment by dfsnow** [comment]
> Did a quick investigation into which parcels are missing census geoids for 2024:
> 
> * 1026 pin10s are missing lat/lon so can't be spatially joined to census
> * 22 pin10s are in the lake, here's a sample:
> 
> image

@wrridgeway I'm guessing this is also the cause of other missing census geographies? (e.g. null congressional district in 2022 sometimes, even though other census geos are filled?)

**Review by dfsnow — APPROVED** [review]
This code changes here look fine @wrridgeway, esp. given that it's a static set of files we're pulling from. That said, let's give the output another once-over here just in case, considering how important this will be for the SoT reporting.

**Comment by wrridgeway** [comment]
I double checked this data, looking at 2016 which has the most null geoids (~12,000), and the vast majority is due to NA lat/long. Only about 800 are missing geoids from the spatial join, and those are from the same lake coast issue.

**Pull request merged**